core/lib/multivm/src/versions/vm_latest/tests/evm_simulator.rs

```diff
@@ -1483,6 +1483,57 @@ fn test_basic_jumpi_vectors() {
     );
 }
 
+#[test]
+fn test_basic_environment3_vectors() {
+    // Here we just try to test some small EVM contracts and ensure that they work.
+    // gasprice
+    let evm_output = test_evm_vector(
+        vec![
+            // push1 32
+            hex::decode("60").unwrap(),
+            hex::decode("20").unwrap(),
+            // push1 16
+            hex::decode("60").unwrap(),
+            hex::decode("10").unwrap(),
+            // gasprice
+            hex::decode("3A").unwrap(),
+            // push32 0
+            hex::decode("7F").unwrap(),
+            H256::zero().0.to_vec(),
+            // sstore
+            hex::decode("55").unwrap(),
+        ]
+        .into_iter()
+        .concat(),
+    );
+    assert_eq!(evm_output, 250000000.into());
+
+    // TODO test OP_CODECOPY
+
+    // codesize
+    let evm_output = test_evm_vector(
+        vec![
+            // push1 32
+            hex::decode("60").unwrap(), // 1 byte
+            hex::decode("20").unwrap(), // 1 byte
+            // push1 16
+            hex::decode("60").unwrap(), // 1 byte
+            hex::decode("10").unwrap(), // 1 byte
+            // gasprice
```

Comment:
This comment should be codesize (the op is ok)

Reply:
Fixed, thanks